### PR TITLE
Update dashboard CSS

### PR DESCRIPTION
**Pull request by shtukas**
... to make it look like the Quiz Builder.

### CODE REVIEW

**Comment by shtukas** [comment]
picture 1

**Comment by shtukas** [comment]
picture 1

**Comment by shtukas** [comment]
There is a remaining little CSS bug, but I need to do some backend work now to be able to display correct values for all the fields, before I come back to the dashboard display.

**Comment by shtukas** [comment]
Note that presence is currently broken

picture 3

File: explainer-server/app/models/ExplainerStore.scala [hunk 1]
```diff
@@ -13,22 +13,17 @@ import scala.concurrent.ExecutionContext.Implicits.global
...
 
 object ExplainerStore {
+
   val defaultCredentialsProvider = new AWSCredentialsProviderChain(
     new ProfileCredentialsProvider("composer"),
     new InstanceProfileCredentialsProvider
   )
 
-  val dynamoDBClient: AmazonDynamoDBAsyncClient  = new AmazonDynamoDBAsyncClient(defaultCredentialsProvider).withRegion(EU_WEST_1)
+  val dynamoDBClient: AmazonDynamoDBAsyncClient = new AmazonDynamoDBAsyncClient(defaultCredentialsProvider).withRegion(EU_WEST_1)
```
Comment (LATaylor-guardian) [inline explainer-server/app/models/ExplainerStore.scala:22]: Region should generally be configurable rather than hardcoded.

Comment (shtukas) [inline explainer-server/app/models/ExplainerStore.scala:22]: Yes. Added to the todo list...

**Comment by shtukas** [comment]
Going to close this PR because the display of an explainer is broken and I made another one, cleaner, which works.

**Pull request closed without merging**